Compiler/General.py

```diff
@@ -858,6 +858,41 @@ def get_op_between_literals_code(op_token):
 
         return code
 
+    elif op == ">>":
+        # a, b, c, x, y, z
```

Comment:
i think this is code duplication of get_divmod_code function, maybe we should call this function here?

Reply:
The code is made so it runs a optimized div 2 while b != 0

If we replace the entire code with divmod we would have to use `1 << b` and then use the result in divmod.

Reply:
You're right :)